src/Wpf.Ui.Demo/Views/Windows/SettingsWindow.xaml

```diff
@@ -5,6 +5,7 @@
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:local="clr-namespace:Wpf.Ui.Demo.Views.Windows"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:navigation="clr-namespace:Wpf.Ui.Controls.Navigation;assembly=Wpf.Ui"
```

Comment:
Just `xmlns:ui` and a new entry in `AssemblyInfo.cs`.

```c#
[assembly: XmlnsDefinition("http://schemas.lepo.co/wpfui/2022/xaml", "Wpf.Ui.Controls.Navigation")]
```

https://github.com/lepoco/wpfui/blob/6e8739676867ecce82a7913ed35c2bb63a59672f/src/Wpf.Ui/Properties/AssemblyInfo.cs#L14